tools/run_tests/stress_test/README.md

```diff
@@ -0,0 +1,72 @@
+﻿Running Stress tests on Google Container Engine
+=======================================
+
+### **Glossary**:
+* GCP: Google Cloud Platform
+* GCE: Google Compute Engine
+* GKE: Google Container Engine
+* GCP console: https://console.cloud.google.com
+
+### **Setup Instructions**
+#### *On GCP:*
+1. Create a GCP account (if you haven’t already) at https://cloud.google.com 

```

Comment:
If the audience of this doc can be someone who never used GCP, it'll be nice to add more instructions besides the links to minimize the time they spend on setup.